test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -445,6 +445,54 @@ private void CrossValidationOn(string dataPath)
             Console.WriteLine(microAccuracies.Average());
         }
 
+        [Fact]
+        public void SdcaLogisticRegression()
```

Comment:
>SdcaLogisticRegression [](start = 20, length = 22)

We keep tests in this file to make sure our Cookbook stays in good shape. Can you move this test somewhere else, or add this sample to cookbook?

Reply:
It is used for testing an example, SDCACalibrated.cs.

---
In reply to: [256211777](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256211777) [](ancestors = 256211777)

Reply:
Where is difference between cookbook and samples. Do you plan to add this case to cookbook? If not, move it to some other place.
We have plenty of places where we test learners, it doesn't have to be part of cookbook

---
In reply to: [256220045](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256220045) [](ancestors = 256220045,256211777)

Reply:
By cookbook I mean this one: [https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md)

---
In reply to: [256220468](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256220468) [](ancestors = 256220468,256220045,256211777)